### DISCUSSION TITLE
Provide the application with the ability to visualize the location of the devices on a floorplan

### DISCUSSION THREAD

**Discussion by lexcorp** [seriousm4x/UpSnap#461] [Feature requests]
I really liked this web application, I was impressed by the ease of installing it and the access to the database, everything integrated is great. I congratulate you, you did a great job.

I suggest adding the option to add a floorplan to graphically show the location of the PCs. I share an example:

https://github.com/seriousm4x/UpSnap/assets/5751288/d0bd1b8a-ed68-4d70-8e08-f8d85f8c5edb

The display in the form of cards is great and the detail of the button that indicates whether the device is on or off seems fabulous to me. But I think that an ideal complement would be to provide the application with the ability to visualize the location of the devices on a floorplan.

![home-dark](https://github.com/seriousm4x/UpSnap/assets/5751288/a251f134-eda8-406b-b81e-5d6e10fc3eaa)

![floorplan](https://github.com/seriousm4x/UpSnap/assets/5751288/2f7f6631-af5f-4542-901b-6d336c276779)

**Comment by seriousm4x** [comment]
Thanks for the suggestion. I kinda like the idea behind this, but I think it's a little bit too abstract.
Also I'm not sure on how this would work on mobile well enough.

**Comment by lexcorp** [comment]
Here another example:

![0B528C2F-E185-4321-8944-EC9871BA58C3](https://github.com/seriousm4x/UpSnap/assets/5751288/33e519db-21ea-4ef3-860a-a5a6e41fb7e1)

https://youtube.com/playlist?list=PLNaeA0iHcKdQ5T4ZBqRtoYvHaCglP-c1D&si=ulQVrPO32_bCa-VT
